package.json

```diff
@@ -36,7 +36,9 @@
         "ws": "^8.13.0",
         "yaml": "^2.3.4",
         "yargs": "^17.7.1",
-        "yauzl": "^2.10.0"
+        "yauzl": "^2.10.0",
+        "@aws-sdk/client-bedrock-runtime": "^3.515.0",
```

Comment:
it isn't. bedrock has a rest api

Reply:
Yes, Bedrock do have a REST API, but I'd like to recommand using Bedrock SDK, cause: 
- Bedrock has its own complicate authN mechanism(i.e. through SigV4), 
- Customized wrapper over streaming data instead of SSE

There might be much more coding effort to do with REST API, thus offical SDK is a better choice.